independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -1331,615 +1376,313 @@
-                      "description": "HubSpot - email open"
+                      "description": "Notes - last contacted."
                     },
-                    "hs_email_optout": {
-                      "title": "HubSpot email optout",
+                    "notes_last_updated": {
+                      "title": "Notes last updated",
                       "type": "string",
-                      "description": "HubSpot - email optout"
+                      "description": "Notes - last updated."
                     },
-                    "hs_email_optout_12592317": {
-                      "title": "HubSpot email optout 12592317",
+                    "notes_next_activity_date": {
+                      "title": "Notes next activity date",
                       "type": "string",
-                      "description": "HubSpot - email optout 12592317"
+                      "description": "Notes - next activity date."
                     },
-                    "hs_email_quarantined": {
-                      "title": "HubSpot email quarantied",
+                    "num_associated_contacts": {
+                      "title": "Number of associated contacts",
                       "type": "string",
-                      "description": "HubSpot - email quarantined"
+                      "description": "The number of associated contacts."
                     },
-                    "hs_email_quarantined_reason": {
-                      "title": "HubSpot email quarantied reason",
+                    "num_contacted_notes": {
+                      "title": "Number of contacted notes",
                       "type": "string",
-                      "description": "HubSpot - email quarantined reason."
+                      "description": "The number of contacted notes"
                     },
-                    "hs_email_replied": {
-                      "title": "HubSpot email replied",
+                    "num_notes": {
+                      "title": "Number of notes",
                       "type": "string",
-                      "description": "HubSpot - email replied"
+                      "description": "The number of notes"
                     },
-                    "hs_email_sends_since_last_engagement": {
-                      "title": "HubSpot email sends since last engagement",
+                    "pipeline": {
+                      "title": "Pipeline",
                       "type": "string",
-                      "description": "HubSpot - email sends since last engagement."
-                    },
-                    "hs_emailconfirmationstatus": {
-                      "title": "HubSpot email confirmation  status",
+                      "description": "Pipeline."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/crm/v3/objects/products": {
+      "get": {
+        "responses": {
+          "default": {

```

Comment:
@ryanedshannon it is not allowed to have a default response only. Please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses). please fix all responses in the swagger file.

Reply:
Hello @ryanedshannon, can you please fix the comment added by Amjed above.

Reply:
Hello @vmanoharas, I think this should already be resolved, I don't see any other responses left with this issue.